main.py

```diff
@@ -9,7 +9,13 @@
 
 BOT_TOKEN = getenv('BOT_TOKEN')
 CHANNEL_ID = getenv('CHANNEL_ID')
-LOG_LEVEL = getenv('LOG_LEVEL')
+
+# LOG_LEVEL can either be an int or a string
+envLog_Level = getenv('LOG_LEVEL')
```

Comment:
This is details but this var name is kind of weird :thinking: you're mixing CamelCase and snake_case, you can name it `LOG_LEVEL` and just overwrite it in case it is numeric! this will also save a useless else case 3 line below

Reply:
fixed by da4b8b33bf710839a20a953e4d5c545cae578565